modules/account-lib/src/mpc/curves/secp256k1.ts

```diff
@@ -0,0 +1,56 @@
+import { publicKeyCombine } from 'secp256k1';
+import { bigIntFromBufferBE, bigIntToBufferBE } from '../util';
+import { modulo } from '../../utils/helpers';
+import BaseCurve from './baseCurve';
+import * as secp from '@noble/secp256k1';
+
+const order = secp.CURVE.n;
+
+export class Secp256k1Curve implements BaseCurve {
+  scalarRandom(): bigint {
+    return bigIntFromBufferBE(Buffer.from(secp.utils.randomPrivateKey()));
+  }
+
+  scalarAdd(x: bigint, y: bigint): bigint {
+    return bigIntFromBufferBE(Buffer.from(secp.utils.privateAdd(modulo(x), modulo(y).toString(16))));
+  }
+
+  scalarSub(x: bigint, y: bigint): bigint {
+    return modulo(x - y);
+  }
+
+  scalarMult(x: bigint, y: bigint): bigint {
+    return modulo(x * y);
+  }
+
+  scalarReduce(s: bigint): bigint {
+    return modulo(s);
+  }
+
+  scalarNegate(s: bigint): bigint {
+    return bigIntFromBufferBE(Buffer.from(secp.utils.privateNegate(modulo(s))));
+  }
+
+  scalarInvert(s: bigint): bigint {
+    return bigIntFromBufferBE(Buffer.from(secp.utils.privateInvert(bigIntToBufferBE(modulo(s), 32))));
```

Comment:
https://github.com/paulmillr/noble-secp256k1/commit/450ebc76ef73ff375c8ce8252468d3a04e87c7f5

noble-secp actually ended up prefering to export `invert` directly as you had originally proposed. Sorry about that! I've updated the `bitgo` branch against his latest `main` so you can sync. up.